.env

```diff
@@ -0,0 +1,10 @@
+# Database credentials
```

Comment:
@tony-tang-aligent it's good practice here to have something like a `.env.example` file with a suggested configuration, rather than a `.env` file inside your repo.  Assume for a moment this is a live website, by having a commited `.env` file configuration on a server could be overwritten by this file on a `git pull`

Reply:
So basically I should use the .env.examaple as a template thingy, then I should have a seperate .env file which follows the template and I should not commit my .env file to the repo, I am getting it correctly?